### PR TITLE
Set port

### PR DESCRIPTION
**Pull request by aviau**
based on #37

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -52,7 +52,7 @@ nodetest-CI: node_modules .node-build-image eslint flow
 
 .PHONY: gobuild
 gobuild: .golang-build-image
-	$(docker_run_go) bash -c "go get -v ./... && go build ./... && go build ./.../cmd/..."
+	$(docker_run_go) bash -c "go get -v ./... && go build ./... && go build github.com/lanets/floorplanets/backend/cmd/floorplanets"
```
Comment (aviau) [inline Makefile]: The older pattern wouldn't match because we added packages.

**Comment by zergov** [comment]
@aviau conflicts

**Comment by aviau** [comment]
fixed

File: Makefile [hunk 2]
```diff
@@ -64,7 +64,7 @@ run-floorplanets: floorplanets
 
 FLOORPLANETS_SOURCES := $(shell find backend -name '*.go') Makefile
 floorplanets: .golang-build-image $(FLOORPLANETS_SOURCES)
-	$(docker_run_go) bash -c "go get -v ./... && go build ./... && go build ./.../cmd/..."
+	$(docker_run_go) bash -c "go get -v ./... && go build ./... && go build github.com/lanets/floorplanets/backend/cmd/floorplanets"
```
Comment (aviau) [inline Makefile]: The older pattern wouldn't match anymore when adding sub-packages in cmd

**Pull request merged**